scripts/dts/extract/globals.py

```diff
@@ -350,6 +375,15 @@ def extract_controller(node_path, prop, prop_values, index,
             continue
 
         cell_parent = phandles[elem[0]]
+
+        if 'gpio-map' in reduced[cell_parent]['props']:
+            # parent is a 'gpio nexus node' (ie has gpio-map)
+            try:
+                # controller is to be found in the map, using elem[1] as index
+                cell_parent = gpio_map_lookup(cell_parent, elem[1])
+            except:
```

Comment:
A bare `try: ... except:` is usually a bad idea, because it will hide stuff like references to undefined Python variables as well (those throw exceptions).

Catch `Exception` instead, or (better) do a more explicit check for whatever this is checking, without exceptions.

Reply:
Yeah, so actually this is the result of a late refactoring and this not chacking anything good anymore. Let me rework that.